compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -637,8 +637,9 @@ static LogicalResult
 setAttentionVectorDistributionConfig(IREE::GPU::TargetAttr target,
                                      mlir::FunctionOpInterface entryPoint,
                                      IREE::LinalgExt::AttentionOp op) {
-  if (target.getWgp().getMma().empty())
+  if (target.getWgp().getMma().empty()) {
```

Comment:
can we get rid of the un-necessary changes such as additions of these brackets

Reply:
Fair enough